src/js/components/PlacementConstraintsFrameworkAdapter.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import Batch from "#SRC/js/structs/Batch";
+import Transaction from "#SRC/js/structs/Transaction";
+import { deepCopy } from "#SRC/js/utils/Util";
+import TransactionTypes from "#SRC/js/constants/TransactionTypes";
+import PlacementConstraintsPartial
+  from "#SRC/js/components/PlacementConstraintsPartial";
+import JSONSingleContainerParser
+  from "#PLUGINS/services/src/js/reducers/JSONSingleContainerParser";
+import CreateServiceModalFormUtil
+  from "#PLUGINS/services/src/js/utils/CreateServiceModalFormUtil";
+
+export default class PlacementConstriantsFrameworkAdapter extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      batch: new Batch(),
+      appConfig: this.getInitialAppConfig()
+    };
+  }
+
+  getInitialAppConfig() {
+    // parse
```

Comment:
Do you think it's worth keeping this comment. It's pretty obvious, isn't it?

Reply:
just WIP :)